packages/features/bookings/lib/getBookingResponsesSchema.ts

```diff
@@ -111,7 +113,10 @@ function preprocess<T extends z.ZodType>({
       }
       for (const bookingField of bookingFields) {
         const value = responses[bookingField.name];
-        const stringSchema = z.string();
+        let stringSchema = z.string();
+        const minLength = bookingField.minLength ?? 0;
+        const maxLength = bookingField.maxLength ?? HARD_LIMIT_MAX_LENGTH;
+        stringSchema = stringSchema.min(minLength).max(maxLength);
```

Comment:
I think we should do it only for `textarea`. 

Though this brings us to the question what if someone enters a long text in a `text` type field. For that the max limit should be much smaller but that is a separate task.

Reply:
I see @hariombalhara , I can create another issue for this if you want.

Reply:
Thanks. It is sorted now.